### ISSUE TITLE
The following Swift pods cannot yet be integrated as static libraries:

### ISSUE THREAD

**Issue by siddharth9470** [Shobbak/react-native-compressor#46]
The Swift pod `react-native-compressor` depends upon `React-Core`, which does not define modules. To opt into those targets generating module maps (which is necessary to import them from Swift when building as static libraries), you may set `use_modular_headers!` globally in your Podfile, or specify `:modular_headers => true` for particular dependencies.

Above error throws when installing pod for ios

**Comment by nomi9995** [comment]
@siddharth9470 can you tell me the version of your `react-native` and `react-native-compressor`

**Comment by siddharth9470** [comment]
react native version is 0.60.6 and react-native-compressor is 0.5.6

**Comment by nomi9995** [comment]
@siddharth9470 it is not reproducing on my side. can you provide a reproducible example so I can fix it?

**Comment by nomi9995** [comment]
Closing due to inactivity.

**Comment by Liqiankun** [comment]
Any update? I got the same issue. 
```
"react": "16.11.0",
"react-native": "0.62.2",
"react-native-compressor": "^1.5.2"
```

**Comment by Liqiankun** [comment]
I know how to fix it. Just remove the `VideoCompressor.swift` file. `pod install` again.

**Comment by Liqiankun** [comment]
@nomi9995 @alariej Please check #103. It has a reproducible example.

**Comment by Liqiankun** [comment]
Fixed #103. Check it out!
